From 0d4213e04ef8078669f072197071ad8630b00bf6 Mon Sep 17 00:00:00 2001
From: Ryan Brady <rbrady@redhat.com>
Date: Tue, 11 Mar 2014 20:06:54 -0400
Subject: [PATCH] Remove Duplicate Rabbit Parameters

This patch removes the duplicate rabbit username and password
parameters.  These parameters cause a conflict when running
merge.py because they already exist in the overcloud template.

Change-Id: I01d386577734c7c516a477e1e402882294ac4376

Conflicts:
	block-storage.yaml
---
 block-storage.yaml | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/block-storage.yaml b/block-storage.yaml
index ca56fce..869f7b8 100644
--- a/block-storage.yaml
+++ b/block-storage.yaml
@@ -4,13 +4,10 @@ Parameters:
   BlockStorageImage:
     Type: String
     Default: overcloud-cinder-volume
-  RabbitUserName:
+  OvercloudBlockStorageFlavor:
+    Default: baremetal
+    Description: Flavor for block storage nodes to request when deploying.
     Type: String
-    Default: "guest"
-  RabbitPassword:
-    Type: String
-    Default: "guest"
-    NoEcho: true
 Resources:
   BlockStorageAccessPolicy:
     Type: OS::Heat::AccessPolicy
-- 
1.8.5.3

